### PR TITLE
(Docs) Some more 3.0 changes

### PR DESCRIPTION
**Pull request by hestonhoffman**
- Update 3.0 page with instructions for installing 2.44
- A couple of small edits 

!no-release-note

### CODE REVIEW

File: documentation/configuring_bolt.md [hunk 1]
```diff
@@ -89,14 +85,14 @@ fields](bolt_inventory_reference.md).
...
+is common to all of your projects. You can set most configurable options in Bolt
+at the user level.
 
-The preferred method for setting user-level configuration is to use a
-`bolt-defaults.yaml` file. This file does not allow you to set project-specific
-configuration, such as the path to an inventory file, and is less likely
-to lead to errors where Bolt loads content from another project.
+To set configuration at the user level, use a `bolt-defaults.yaml` file. This
+file does not allow you to set project-specific configuration, such as the path
+to an inventory file, and is less likely to lead to errors where Bolt loads
```
Comment (beechtom) [inline documentation/configuring_bolt.md:94]:
```suggestion
file does not allow you to set project-specific configuration, such as the modulepath,
and is less likely to lead to errors where Bolt loads
```

3.0 gets rid of the `inventoryfile` config option

File: documentation/configuring_bolt.md [hunk 2]
```diff
@@ -117,10 +113,10 @@ proxy, the number of threads Bolt should use when connecting to targets, or
 setting credentials for connecting to PuppetDB. You can set most configurable
 Bolt options at the system level.
 
-The preferred method for setting user-level configuration is to use a
-`bolt-defaults.yaml` file. This file does not allow you to set project-specific
-configuration, such as the path to an inventory file, and is less likely
-to lead to errors where content from another project is loaded.
+To set configuration at the user level, use a `bolt-defaults.yaml` file. This
+file does not allow you to set project-specific configuration, such as the path
+to an inventory file, and is less likely to lead to errors where content from
```
Comment (beechtom) [inline documentation/configuring_bolt.md:118]:
```suggestion
file does not allow you to set project-specific configuration, such as the modulepath,
and is less likely to lead to errors where content from
```

File: documentation/configuring_bolt.md (hunk 1 above)
Comment (hestonhoffman) [reply documentation/configuring_bolt.md:94]: Ah right, I thought I'd removed this already

File: documentation/upgrading_to_bolt_3.md [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
+need to know if you're upgrading from an earlier version of Bolt.
 
 For an exhaustive list of the things we've changed, see: 
-- [Changes coming in Bolt 3.0](developer_updates.md#changes-coming-in-bolt-30).
-- [Bolt 3.0 Changelog](TODO:insert-link-here)
+- [Changes coming in Bolt 3.0](developer_updates.md#changes-coming-in-bolt-30)
+- [Bolt 3.0 Changelog](https://pup.pt/bolt-3)
+
+If you're not ready to make the leap to Bolt 3.0, you can still download the
+last release in the Bolt 2 series by specifying the version (2.40.1) in your
```
Comment (lucywyman) [inline documentation/upgrading_to_bolt_3.md:11]:
```suggestion
last release in the Bolt 2 series by specifying the version (2.44.0) in your
```

**Review by beechtom — APPROVED** [review]

File: documentation/upgrading_to_bolt_3.md [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
-- [Bolt 3.0 Changelog](TODO:insert-link-here)
+- [Changes coming in Bolt 3.0](developer_updates.md#changes-coming-in-bolt-30)
+- [Bolt 3.0 Changelog](https://pup.pt/bolt-3)
+
+If you're not ready to make the leap to Bolt 3.0, you can still download the
+last release in the Bolt 2 series by specifying the version (2.44.0) in your
+package manager. For Homebrew, use `brew install --cask puppet-bolt@2`. You can
+also download the relevant
+[MacOS](https://downloads.puppet.com/mac/puppet6/10.15/x86_64/) and
+[Windows](http://downloads.puppet.com/windows/puppet6/) installers.
```
Comment (beechtom) [inline documentation/upgrading_to_bolt_3.md:15]:
```suggestion
[macOS](https://downloads.puppet.com/mac/puppet-tools/10.15/x86_64/) and
[Windows](http://downloads.puppet.com/windows/puppet-tools/) installers.
```

We should point to the `puppet-tools` repos, not `puppet*`.

**Review by beechtom — CHANGES_REQUESTED** [review]

**Review by beechtom — COMMENTED** [review]
We should update the links to the MSI installers and DMGs in the install docs so they also point at the `puppet-tools` repo: https://github.com/puppetlabs/bolt/blob/main/documentation/bolt_installing.md

**Review by beechtom — APPROVED** [review]

**Review by lucywyman — APPROVED** [review]

**Pull request merged**